bitnami/kafka/Chart.yaml

```
@@ -40,4 +40,4 @@ maintainers:
 name: kafka
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kafka
-version: 29.1.2
+version: 29.1.3
```

Please bump a minor version

```suggestion
version: 29.2.0
```

We use semver.org for versioning, which translates into:
- Major version for non-backward compatible changes.
- Minor version for new values.
- Patches for bug fixes and enhancements for existing features.